www/docs/reactjs/useContext.mdx

```diff
@@ -95,6 +95,49 @@ These are the helpers you'll get access to via `useContext`. The table below wil
 | `setInfiniteData`   | [`queryClient.setInfiniteQueryData`](https://tanstack.com/query/v4/docs/reference/QueryClient#queryclientsetquerydata)           |
 | `getInfiniteData`   | [`queryClient.getInfiniteData`](https://tanstack.com/query/v4/docs/reference/QueryClient#queryclientgetquerydata)                |
 
+### ❓ The function I want isn't here!
+
+`@tanstack/react-query` has a lot of functions that we haven't put in the tRPC context yet. If you need a function that isn't here, feel free to [open a feature request](https://github.com/trpc/trpc/issues/new/choose) requesting it.
+
+In the meantime, you can import and use the function directly from `@tanstack/react-query`. We provide a `getQueryKey` helper on routers and procedures so that you can easily provide the native function the correct query key.
+
+```tsx
+function getQueryKey(
+  input: TInput,
+  type?: QueryType;
```

Comment:
can we document the default value of `type` here? Looking at the code, I think it defaults to `"any"` ?

Reply:
Yes definetely should be documented